templates/docker-container-to-aks.properties.json

```diff
@@ -6,6 +6,8 @@
     { "name" : "kubernetesConnectionId", "type": "create:azureKubernetesEndpoint", "required": "true"},

```

Comment:
Rename this to `kubernetesConnection` instead, as you're accessing it as an object.